relayer-cli/src/commands/tx/client.rs

```diff
@@ -214,18 +215,26 @@ pub struct TxUpgradeClientCmd {
         help = "Identifier of the client to be upgraded"
     )]
     client_id: ClientId,
+
+    #[clap(
+        long = "upgrade-height",
+        required = true,
+        value_name = "HOST_TARGET_UPGRADE_HEIGHT",
```

Comment:
This should be `"REFERENCE_UPGRADE_HEIGHT"`, as the reference chain is the one performing the upgrade at height `upgrade-height`.